source/Cosmos.System2/Graphics/VBECanvas.cs

```diff
@@ -369,6 +405,93 @@ public override void DrawImage(Image aImage, int aX, int aY)
             Global.mDebugger.SendInternal("Done");
         }
 
+        /// <summary>
+        /// Draw image with alpha channel.
+        /// </summary>
+        /// <param name="image">Image to draw.</param>
+        /// <param name="x">X coordinate.</param>
+        /// <param name="y">Y coordinate.</param>
+        /// <exception cref="Exception">Thrown on memory access violation.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown on fatal error.</exception>
+        public override void DrawImageAlpha(Image image, int x, int y)
+        {
+            Global.mDebugger.SendInternal($"Drawing image of size {image.Width}x{image.Height} array size {image.rawData.Length} with alpha chanel...");
+            for (int _x = 0; _x < image.Width; _x++)
+            {
+                for (int _y = 0; _y < image.Height; _y++)
+                {
+                    _VBEDriver.SetVRAM((uint)GetPointOffset(y + _y, x + _x), (byte)image.rawData[_x + _y * image.Width]);
```

Comment:
Isnt this ignoring the alpha data of the image? Isnt the implementation the same as in the base case.